src/client/components/Navigation/NavGeo/AgreementLevelSelector/AgreementLevelSelector.scss

```diff
@@ -0,0 +1,42 @@
+@import 'src/client/style/partials';
+
+.geo-agreement-level-selector__container {
+  display: flex;
+  justify-content: space-between;
+
+  .geo-agreement-level-selector__checkbox-container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-xxs;
+
+    &.disabled {
+      color: $text-disabled;
+    }
+
+    > label {
+      font-size: $font-s;
+    }
+  }
+
+  .geo-agreement-level-selector__checkbox {
+    appearance: none;
+    border-radius: 3px;
+    border: 1px solid #949495;
+    cursor: pointer;
+    height: $spacing-s;
+    width: $spacing-s;
+
+    &:checked {
+      background-color: #262627;
+      border-color: #5f5f5f;
```

Comment:
I tried using existing colors for this and other instances in this file but it looked off. Should we add new color variables for these?